app/Http/Controllers/Auth/EmailConfirmationController.php

```diff
@@ -21,7 +21,7 @@ public function send()
         if (Auth::user()->isConfirmed()) {
             $this->error('auth.confirmation.already_confirmed');
         } else {
-            $this->dispatchNow(new SendEmailConfirmation(Auth::user()));
+            $this->dispatch(new SendEmailConfirmation(Auth::user()));
 
             $this->success('auth.confirmation.sent', Auth::user()->emailAddress());
```

Comment:
Should we replace `auth.confirmation.sent` with `auth.confirmation.queued`?

Reply:
Nah, that's ok.